.circleci/config.yml

```diff
@@ -188,11 +190,12 @@ commands:
               - equal: [ *windows_test_executor, << parameters.platform >> ]
           steps:
             - run:
-                name: Create bash aliases
+                name: Extend Bash profile for Windows
                 command: |
                   echo 'alias find=/bin/find' >> "$BASH_ENV"
                   echo 'alias sort=/bin/sort' >> "$BASH_ENV"
                   echo 'export EXECUTABLE_SUFFIX=".exe"' >> "$BASH_ENV"
+                  echo 'export PATH="$HOME/AppData/Local/mise/shims:$HOME/scoop/apps/mise/current/bin:$HOME/scoop/shims:$PATH"' >> "$BASH_ENV"
```

Comment:
Similarly would suggest mise env instead of manually writing the line. You can always add more bits to PATH with mise itself.

Reply:
Windows just wasn't playing ball here immediately, but maybe it was because i was using [activate](https://mise.jdx.dev/cli/activate.html#mise-activate).  i ended up just copying and pasting something that just worked when I was SSH'd in, but I'll try again in another PR.  Mise basically does say this is fine in their docs and that they'd never change it, but still wouldn't mind the abstraction. Thanks!

Reply:
I missed that this was the window section... I might still be right though, idk